### ISSUE TITLE
Qt5 fullscreen - Only rendering in part of window

### ISSUE THREAD

**Issue by Atnas1010** [vispy/vispy#1542]
I made a program that shows an image in fullscreen, but when I use the Qt5 backend, it shows the full image, but utilizes only part of the screen (to the top and right of the image, there's only white)

If I change the backend to glfw, it uses the whole display as expected.

Following is a MWE:
```
import numpy as np
from vispy import app, gloo
c = app.Canvas(keys='interactive', fullscreen=0, app='pyqt5')

vert_shader = """
attribute vec2 a_position;
varying vec2 v_texcoord;
void main () {
    // Compute texture coordinates
    v_texcoord = vec2(a_position.x, a_position.y)*.5+.5;
    gl_Position = vec4(a_position, 0.0, 1.0);
}"""

frag_shader = """
uniform sampler2D u_texture;
varying vec2 v_texcoord;
void main() {
    gl_FragColor = texture2D(u_texture, v_texcoord);
    gl_FragColor.a = 1.0;
}"""

program = gloo.Program(vert_shader, frag_shader)

positions = np.array([[1.0, -1.0], [-1.0, -1.0],
                      [1.0, 1.0], [-1.0, 1.0]], np.float32)
program['a_position'] = positions

@c.connect
def on_draw(event):
    gloo.clear((1, 1, 1, 1))
    program.draw('triangle_strip')

program['u_texture'] = gloo.Texture2D(np.array([[0, 1], [1, 0]], np.float32))
c.show()
c.app.run()
```

**Comment by djhoese** [comment]
What version of vispy? Have you tried PyQt4? Are the results different?

**Comment by Atnas1010** [comment]
Python 3.6.0
Vispy 0.5.3

It's the same result under PyQt4 (a bit different part of the image not used, but still top and right parts where it's not used)

**Comment by djhoese** [comment]
What operating system? Could you try installing vispy from the github repository master branch and see if that helps?

```
pip install git+https://github.com/vispy/vispy.git
```

**Comment by Atnas1010** [comment]
I was using Windows 10, but fixing my paths to compile vispy was hard so:
So under Ubuntu 18.04, I get the same behavior with QtPy5. Tried with vispy from master branch and 0.5.3, and got same results.
The only difference is that is uses a much smaller part of the screen, but still top and right part not used (but this is also on another computer, so maybe it's just the screen resolution making it vary)

**Comment by djhoese** [comment]
Do you get any weird behavior from the vispy examples? There are a couple that open full screen so maybe those would be good for us to use as a starting point for testing.

**Comment by Atnas1010** [comment]
Correct me if I'm wrong, but the only fullscreen example I could find was 
vispy/examples/demo/scene/oscilloscope.py

When I run the example, it uses my full screen.
![initial](https://user-images.githubusercontent.com/11504118/47876732-67e8ce00-ddf0-11e8-9413-7faf5bb7a895.jpg)
If I click on the rendering, it moves into the lower left, it moves into the lower left corner and stays there. This is just like the problem I'm experiencing, except that the unused area is black here, and white in my posted example.
![afterclicking](https://user-images.githubusercontent.com/11504118/47876731-67e8ce00-ddf0-11e8-9dfb-c123f5f5665f.jpg)

**Comment by djhoese** [comment]
You may be right that that is the only one that opens up in fullscreen by default. Others use the default interactive key bindings so you can make them fullscreen (F11 maybe?).

Anyway, I *swear* I fixed this in github master. I'll have to try this later. I'm currently on vacation so not sure when I'll get to it.

**Comment by djhoese** [comment]
Just tested this on OSX with the master branch and it behaved as expected. Any idea if you have a HiDPI display? I wonder if the Qt library isn't properly built and isn't detecting the HiDPI display properly.

**Comment by Atnas1010** [comment]
I just tested the oscilloscope on my ubuntu machine (no display scaling) and it worked as properly on 0.5.3 and master.
After updating my windows machine to master, the oscilloscope.py is working fine.
My original posted code still gives weird behavior on ubuntu and windows with both 0.5.3 and master.
My ubuntu machine uses a single display with 100% display scaling.
My windows machine uses two displays with 100% on the main, and 150% display scaling on the secondary. I also tried with 100% on both, but get the same weird behavior from my initial code.
